src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -35,110 +55,193 @@ public class AdjournCaseAboutToSubmitHandlerTest {
     @Mock
     private CaseDetails<SscsCaseData> caseDetails;
 
+    @Mock
+    private ListAssistHearingMessageHelper hearingMessageHelper;
+
+    @SuppressWarnings("unused")
+    @Mock
     private PreviewDocumentService previewDocumentService;
+
     private SscsCaseData sscsCaseData;
+    private AutoCloseable autoCloseable;
 
-    @Before
+    @BeforeEach
     public void setUp() {
-        openMocks(this);
-        previewDocumentService = new PreviewDocumentService();
-        handler = new AdjournCaseAboutToSubmitHandler(previewDocumentService);
+        autoCloseable = openMocks(this);
```

Comment:
You can remove this now (and the tearDown method) and use (the better) annotation on the class:
@RunWith(SpringRunner.class)